airbyte-webapp/package.json

```diff
@@ -68,7 +69,7 @@
     "react-paginate": "^8.1.3",
     "react-query": "^3.39.1",
     "react-reflex": "^4.0.9",
-    "react-router-dom": "^6.3.0",
+    "react-router-dom": "6.3.0",
```

Comment:
:information_source: Need to fix that version for now, due to `useBlocker`. This uses unstable code that's no longer there in the most recent react-router-dom version. We should rewrite that code to the newest `unstable_useBlocker` code, but not as part of this PR :D

Reply:
I did explicitally NOT mean to update the version. The newer version breaks our `useBlocker` hook, since that was build using internal APIs that are not available in newer versions anymore. So I set a fixed version for now (since `pnpm` would have pulled in a newer one). I've created https://github.com/airbytehq/airbyte/issues/22305 to track rewriting this code and updating.

Reply:
Ah okay, I had just misunderstood this then. That makes sense!